.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -1,30 +1,70 @@
 ---
 name: Report a bug
-about: Something with go-ethereum is not working as expected
+about: Something with bor client is not working as expected
 title: ''
 labels: 'type:bug'
 assignees: ''
 ---
 
-#### System information
+Our support team has aggregated some common issues and their solutions from past which are faced while running or interacting with a bor client. In order to prevent redundant efforts, we would encourage you to have a look at the [FAQ's section](https://docs.polygon.technology/docs/faq/technical-faqs) of our documentation mentioning the same, before filing an issue here. In case of additional support, you can also join our [discord](https://discord.com/invite/zdwkdvMNY2) server
 
-Geth version: `geth version`
-OS & Version: Windows/Linux/OSX
-Commit hash : (if `develop`)
+<!--
+NOTE: Please make sure to check of any addresses / private keys / rpc url's / IP's before sharing the logs or anything from the additional information section (start.sh or heimdall config).
+-->
 
-#### Expected behaviour
+#### **System information**
 
+Bor client version: [e.g. v0.2.16]
 
-#### Actual behaviour
+Heimdall client version: [e.g. v0.2.10]
```

Comment:
Could we add instructions how a user could check versions of services?

Reply:
Sure, done!